translations/es.json

```diff
@@ -99,9 +103,11 @@
   "label.about": "Sobre UDOIT",
   "label.reports": "Reportes",
   "label.report_history": "Historial",
-  "label.no_results_header": "No results found",
+  "label.no_results_header": "No se han encontrado resultados",
```

Comment:
This works but "No se encontraron resultados" sounds better because the former translates to "Haven't found any results" which kinda sounds like the search continues.